PerfectWeek/src/app/pages/Connection/connection.html

```diff
@@ -17,7 +17,7 @@
         </div>
 
         <div class="text-center">
-          <button [disabled]="!registrationForm.valid" class="btn btn-default waves-light" mdbWavesEffect>S'inscrire</button>
+          <button [disabled]="!connectionForm.valid" class="btn btn-default waves-light" mdbWavesEffect>S'inscrire</button>
```

Comment:
You can use ".invalid" instead of "!.valid"
It is more clear about your intentions

Not a huge change tho